client/src/components/AnimeList/AnimeList.js

```diff
@@ -1,18 +1,34 @@
-import React from "react";
+import React from 'react'
+const AnimeList = ({ animelist,setAnimeInfo,animeComponent,handleList}) => {
+  const AddToList=animeComponent;
```

Comment:
I'm confused why you're doing this. Is this a static component or does it vary?

If it is static, why not just import the component you need?

If it is not static, you don't need to reassign it to an intermediate variable before using it on line 20.